unit_tests/linux/mock_os_layer.cpp

```diff
@@ -141,7 +141,7 @@ int ioctl(int fd, unsigned long int request, ...) throw() {
     int res;
     va_list vl;
     va_start(vl, request);
-
+	
```

Comment:
please remove extra white spaces in this line, it does not meed clang-format rules.